diff --git a/build/index.js b/build/index.js
index 1e1164d52b8bcbd6f0ba0bf043cba958409c7df3..45758a9b9847fc02bbdc4d8b4df1f1c98466172e 100644
--- a/build/index.js
+++ b/build/index.js
@@ -33,8 +33,9 @@ module.exports = function stripExports(options) {
             estree_walker_1.walk(ast, {
                 enter: function (baseNode) {
                     var node = baseNode;
-                    if (node && node.range) {
-                        var _a = node.range, start = _a[0], end = _a[1];
+                    if (node && node.start && node.end) {
+                        var start = node.start;
+                        var end = node.end;
                         if (options.sourceMap) {
                             magicString.addSourcemapLocation(start);
                             magicString.addSourcemapLocation(end);
